src/containers/ImageUploader/CreateImage.tsx

```diff
@@ -13,14 +13,13 @@ import { createFormData } from '../../util/formDataHelper';
 import * as imageApi from '../../modules/image/imageApi';
 import { toEditImage } from '../../util/routeHelpers';
 import { License } from '../../interfaces';
-import { NewImageMetadata } from '../../modules/image/imageApiInterfaces';
+import { UpdatedImageMetadata } from '../../modules/image/imageApiInterfaces';
 import { fetchLicenses } from '../../modules/draft/draftApi';
 import { draftLicensesToImageLicenses } from '../../modules/draft/draftApiUtils';
 
-interface Props {
-  history: RouteComponentProps['history'];
-  isNewlyCreated: boolean;
-  showSaved: boolean;
+interface Props extends RouteComponentProps {
+  isNewlyCreated?: boolean;
+  showSaved?: boolean;
 }
 
 const CreateImage = ({ history, isNewlyCreated, showSaved }: Props & RouteComponentProps) => {
```

Comment:
Siden du extender `RouteComponentProps` for Props, tror ikke du trenger den lenger her nede